### ISSUE TITLE
Firebase Translate get Models for translation failure due to SSL issue

### ISSUE THREAD

**Issue by prmacias** [firebase/quickstart-ios#961]
I have been unit testing the firebase translate and it seems the server is having the ssl issue when I download the models. 
Here's the error message -

```
An SSL error has occurred and a secure connection to the server cannot be made
error : Failed to download the model from the server.
```

Here's the func - 

```
func getModels(sourceLanguage : String, targetLanguage : String, handler: @escaping (Bool) -> Void) {
        let sourceLanguage = getTranslateLanguage(languageCode: sourceLanguage)
        let targetLanguage = getTranslateLanguage(languageCode: targetLanguage)
        let options = TranslatorOptions(sourceLanguage: sourceLanguage, targetLanguage: targetLanguage)
        translator = NaturalLanguage.naturalLanguage().translator(options: options)
        
        let conditions = ModelDownloadConditions(
          allowsCellularAccess: false,
          allowsBackgroundDownloading: true
        )
        
        translator.downloadModelIfNeeded(with: conditions) { error in
            guard error == nil else {
                let errorDesc  = error?.localizedDescription ?? "unknown error"
                
                os_log("error : %@", log: self.tag, type: .debug, errorDesc)
                handler(false)
                return
            }
            handler(true)
        }
    }
```

Here's the unit test function - 

```
func testGetModels() {
        let expectation = self.expectation(description: "testGetModels")
        let sourceLanguage = "en"
        let targetLanguage = "es"
        
        sut.getModels(sourceLanguage: sourceLanguage, targetLanguage: targetLanguage) { (result) in
            expectation.fulfill()
            if(!result) {
                fatalError()
            }
        }
        
        waitForExpectations(timeout: 10, handler: nil)

        XCTAssertTrue(sut.isLanguageDownloaded(sourceLanguage))
        XCTAssertTrue(sut.isLanguageDownloaded(targetLanguage))
    }
```

**Comment by morganchen12** [comment]
Is this reproducible in the simulator when not running unit tests?

**Comment by prmacias** [comment]
@morganchen12 

Yes, I placed the code in appdelegate and I'm still getting the same error in my iphone and simulator. A strange thing happened when I used simulator (I was using my iphone) the unit test passed but when I ran the unit tests afterwards and then the error came up and tried it again but not able to resolve the issue.

**Comment by morganchen12** [comment]
Are you able to download files from Firebase Storage on any device on your network?

**Comment by prmacias** [comment]
@morganchen12 

I tried a new simulator and translation in appdelegate is working fine then I ran the unit tests and it failed eventually something wrong with my code somehow. I think it might be related to saved models. I'll review the code - I'll close this issue if I see anything is amiss then I'll reopen this issue. 

Update - 

I found the error - I changed to 10 seconds to 50 seconds to give some time for the unit tests to download the models. 

waitForExpectations(timeout: 50, handler: nil)

Closing this issue.  Thanks!
